core/domain/exp_services.py

```diff
@@ -385,14 +496,26 @@ def get_recently_published_exploration_summaries():
 def get_all_exploration_summaries():
     """Returns a dict with all exploration summary domain objects,
     keyed by their id.
+
+    Returns:
+        A python dict with all ExplorationSummary domain objects.
     """
     return _get_exploration_summaries_from_models(
         exp_models.ExpSummaryModel.get_all())
 
 
 # Methods for exporting states and explorations to other formats.
 def export_to_zip_file(exploration_id, version=None):
-    """Returns a ZIP archive of the exploration."""
+    """Returns a ZIP archive of the exploration.
+
+    Args:
+        exploration_id: exploration id in question.
```

Comment:
exploration_id: str. the id of the exploration to export.
version: ??? or None. If provided, this indicates which version of the exploration to export. Otherwise, the latest version of the exploration is exported.

(Btw, just to be clear, you'll need to figure out what ??? is.)